onyo/lib/onyo.py

```diff
@@ -603,31 +616,34 @@ def get_item_paths(self,
         if depth < 0:
             raise ValueError(f"depth must be greater or equal 0, but is '{depth}'")
 
-        # Note: The if-else here doesn't change result, but utilizes `GitRepo`'s cache:
-        files = self.git.get_files(include) if include else self.git.files
+        exclude = [exclude] if isinstance(exclude, Path) else exclude
+        if not any(self.is_template_path(p) for p in include):
+            # if no template dir is explicitly given, remove the template subdir entirely:
```

Comment:
This is probably the best way to do it for now. But I'm not a big fan of it.

We should keep this in mind as we talk about how we handle templates. Whether they are considered part of the inventory, or if they are an untrusted source, which were are able to do a best-effort query of when it's passed as a path.